vent/coordinator/process_manager.py

```diff
@@ -1,19 +1,67 @@
 import threading
 import time
+import os
+from vent.coordinator.ipc import IPC
+import vent
+from time import sleep
+from vent.coordinator.ipc import IPCCommand, IPCMessage
+import multiprocessing
+
+
+class ChildProcess:
+    def __init__(self, sim_mode, addr, port):
+        self.local_coordinator = vent.coordinator.coordinator.get_coordinator(sim_mode=sim_mode, single_process=True)
+        self.ipc = IPC(False, addr, port)
+        self.receive_thread = threading.Thread(target=self.__child_receive_loop, daemon=True)
+        self.send_thread = threading.Thread(target=self.__child_send_loop, daemon=True)
```

Comment:
I'm confused, why does the process manager have send / receive threads?

Reply:
it looks like the send_loop is just updating sensors, so i don't know whether send/receive is exactly right. I'm also not really sure why process manager needs to be handling any data -- controller -> coordinator -> ui seems right.

Reply:
Changing to RPC solves this issue.